### PR TITLE
add details running on Windows

### PR DESCRIPTION
**Pull request by yimaixinchen**

### CODE REVIEW

**Comment by yimaixinchen** [comment]
this is not running on cmd.exe as ADMINISTRATOR.
d4bf2036b91a42559c5f6918ff6dfbef-60872

this is running on cmd.exe as ADMINISTRATOR.
捕获

File: README.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ Please also make sure your node version is 10+, version lower than 10.x is not s
 
 1. Run `npm install` in the root directory to install the dependencies.
 2. Run `npm run start` in the root directory to start a local server, you will see the website in 'http://localhost:8080'.
+```
+Note:if you clone the code in Windows not Mac .please read details as below.
+if you execute the commands like two steps,you will get exception "UnhandledPromiseRejectionWarning: Error: EPERM: operation not permitted, symlink '2.0.3' -> 'latest'".
```
Comment (QuakeWang) [inline README.md:17]:
```suggestion
If you execute the commands like the two steps above, you will get the exception "UnhandledPromiseRejectionWarning: Error: EPERM: operation not permitted, symlink '2.0.3' -> 'latest'".
```

Comment (QuakeWang) [reply README.md:17]: Does this express it a little more accurately?

File: README.md [hunk 2]
```diff
@@ -12,6 +12,11 @@ Please also make sure your node version is 10+, version lower than 10.x is not s
...
+When you meet this problem.you will run two steps in the cmd.exe as an  ADMINISTRATOR MEMBER.
```
Comment (QuakeWang) [inline README.md:18]:
```suggestion
When meeting this problem. You can run two steps in the cmd.exe as an  ADMINISTRATOR MEMBER.
```

**Comment by yimaixinchen** [comment]
Thanks for your suggestion.I fix it.

File: README.md [hunk 3]
```diff
@@ -12,6 +12,11 @@ Please also make sure your node version is 10+, version lower than 10.x is not s
 
 1. Run `npm install` in the root directory to install the dependencies.
 2. Run `npm run start` in the root directory to start a local server, you will see the website in 'http://localhost:8080'.
+```
+Note:if you clone the code in Windows not Mac .please read details as below.
```
Comment (QuakeWang) [inline README.md:16]:
```suggestion
Note: if you clone the code in Windows, not Mac. Please read the details below.
```

Comment (QuakeWang) [reply README.md:16]: Note the space separators.

**Comment by zhongjiajie** [comment]
Thank @yimaixinchen for the PR, and @QuakeWang for reviewing

**Review by QuakeWang — DISMISSED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -12,6 +12,11 @@ Please also make sure your node version is 10+, version lower than 10.x is not s
 
 1. Run `npm install` in the root directory to install the dependencies.
 2. Run `npm run start` in the root directory to start a local server, you will see the website in 'http://localhost:8080'.
+```
+Note: if you clone the code in Windows, not Mac. Please read the details below.
+If you execute the commands like the two steps above, you will get the exception "UnhandledPromiseRejectionWarning: Error: EPERM: operation not permitted, symlink '2.0.3' -> 'latest'".
+When meeting this problem. You can run two steps in the cmd.exe as an  ADMINISTRATOR MEMBER.
```
Comment (zhongjiajie) [inline README.md:18]:
```suggestion
When meeting this problem. You can run two steps in the cmd.exe as an ADMINISTRATOR MEMBER.
```

**Review by zhongjiajie — APPROVED** [review]
Looking good and thanks @yimaixinchen

**Pull request merged**